Packs/ContentManagement/Scripts/CommitFiles/CommitFiles.py

```diff
@@ -133,6 +135,21 @@ def does_file_exist(branch_name: str, content_file: ContentFile) -> bool:
     return False
 
 
+def does_file_exist_azure_devops(branch_name: str, content_file: ContentFile) -> bool:
+    full_path = Path(content_file.path_to_file, content_file.file_name)
+
+    if str(full_path) in files_path:
```

Comment:
what's `files_path`? I see it was there before, please add a comment here, or change its name, to something clearer